packages/api-openfin/window.js

```diff
@@ -2,4 +2,48 @@ if (!window.ssf) {
   window.ssf = {};
 }
 
-window.ssf.window = (...args) => window.open(...args);
+window.ssf.window = function(url, name, features) {
+  let newWindow;
+  const featureObject = parseFeaturesString(features);
+  const handleError = (error) => console.error('Error creating window: ' + error);
+
+  if (featureObject.child) {
+    newWindow = new fin.desktop.Window({
+      name,
+      url
+    }, () => newWindow.show(), handleError);
+  } else {
+    // UUID must be the same as name
+    const uuid = name;
+    const mainWindowOptions = {
+      autoShow: true
+    };
+
+    const app = new fin.desktop.Application({
+      name,
+      url,
+      uuid,
+      mainWindowOptions
+    }, () => app.run(), handleError);
+
+    // Need to return the window object, not the application
+    newWindow = app.getWindow();
+  }
+
+  return newWindow;
+};
+
+const parseFeaturesString = (features) => {
```

Comment:
Thanks for updating this function in the Electron API - please can these changes also be replicated here?